src/drive/web/modules/views/OnlyOffice/helpers.js

```diff
@@ -58,6 +58,30 @@ export const makeOnlyOfficeIconByClass = fileClass => {
   return iconByClass[fileClass]
 }
 
+export const makeExtByClass = fileClass => {
```

Comment:
why make? Doesn't get smaller and better?

Reply:
Le `get` est pour les document selon la règle `Should access data without making a server side call`
Pour les fonctions on a : 
```
- compute: Create something from other elements
- make: Create something from scratch
```

A la limite ce serait plutôt `compute` ici en fait...